### DISCUSSION TITLE
Switching between 4k & 7k best performances on website

### DISCUSSION THREAD

**Discussion by Wavelation** [ppy/osu#25698] [Ideas] [answered]
Hi, 

I'm a 4k player since a long time and sometimes likes to enjoy myself on some 7k maps. I've been grinding my level at some point but never feel rewarded to see any of my play appearing on my userpage because of the gap of some 500+pp i've earn in 4k.

Since we're able to see how many pp we got in 7k on the website, couldn't be possible to have something like this ? : 

![bestperf](https://github.com/ppy/osu/assets/119364582/abcbc852-0a21-43a1-8313-6d89a08493e5)

As always, thanks for all the work you're doing with osu!

lot of love from france <3

**Comment by Tupidix** [comment]
Please yes

**Comment by verarr** [comment]
All the other keycounts too! (I could literally say the same as the original post, except with 6K instead of 7K). If it becomes too much, you could show only the keycounts in which the player has achieved a relevant amount of pp (>10pp) or score (idk?) or sort them by those values, so the important ones are at the front.

Also, I think I'd like to see my rank only in specific keycounts (stable had this with 4K and 7K), in which case a better place for this _might_ be next to/below these buttons?
![image](https://github.com/ppy/osu/assets/72462464/875ddaa6-6250-4297-818a-d31bad80166d)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu-web/issues/6578 is probably the closest web-side issue for this, nothing can be done about this on the game side
